velox/dwio/parquet/writer/arrow/util/CMakeLists.txt

```diff
@@ -15,4 +15,5 @@
 add_library(velox_dwio_arrow_parquet_writer_util_lib
             Compression.cpp CompressionSnappy.cpp Hashing.cpp Crc32.cpp)
 
-target_link_libraries(velox_dwio_arrow_parquet_writer_util_lib parquet arrow)
+target_link_libraries(velox_dwio_arrow_parquet_writer_util_lib parquet arrow
+                      Snappy::snappy)
```

Comment:
I'm surprised this didn't fail on linux since there is a dependency indeed. Thanks @czentgr for the quick fix!

Reply:
I suspect that on Linux the system include (like `/usr/include`) solves the problem while on Mac these include files are not in the system include path and thus need to be explicitly added via dependency.